### ISSUE TITLE
"read_csv" introduces bugs when treat ICD9_CODE be integer by default. And codes can appear more than once for a single HADM_ID

### ISSUE THREAD

**Issue by airingzhang** [jamesmullenbach/caml-mimic#25]
They both make the top50 statistics changed. And with MIMIC-iii 1.4,  the full label seems not to be 8922 anymore (if the ICD9_CODE is read as str)

**Comment by LeiGong0125Carrot** [comment]
> They both make the top50 statistics changed. And with MIMIC-iii 1.4, the full label seems not to be 8922 anymore (if the ICD9_CODE is read as str)

yes, I also found this, it seems like the correct number should be 9017
